### DISCUSSION TITLE
Checkout Assets to Departments

### DISCUSSION THREAD

**Discussion by ghost** [snipe/snipe-it#10992] [Ideas]
It would be nice if you can checkout assets to departments. In my company we dont check out monitors or pcs to employees. Only personal phones, laptops etc.

**Comment by enbyprogrammer** [comment]
I second this. At the organization I work at we have many assets that would just be assigned to a department rather than a specific user. If it is assigned to a department the ability to essentially add a location within the department onto the asset's assignment could also be useful.

**Comment by BraindamageBE** [comment]
I would love to see the same.
We also have desktops and monitors assigned to departments and not employees.
Assigning it to a location is possible, but then you assign it to a whole warehouse/factory/...
Thanks!

**Comment by tpaljor2010** [comment]
I agree with them. However, it must also be put into consideration that a asset manager at department level will get access to the assets checked out to its department. The asset manager at department level should be able check out assets to users belonging the department. I have not explored if there are possibilities of implementing the company->department->section and SuperAdmin->DepartmentAdmin->DepartmentUser system.

**Comment by sm-bdsc** [comment]
Additionally, being able to check licenses out to department for tracking cost-centres would be great!

**Comment by Akalavolo** [comment]
Hello, is there any development underway for this feature? In my company we also assign certain hardware (screen, docking station, etc.) to the department and not to the user.
